src/Microsoft.DotNet.ProjectModel/RuntimeOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.ProjectModel
+{
+    public class RuntimeOptions
```

Comment:
I am not so sure this is the right thing. RuntimeOptions is open, meaning that it is not limited to these two options. Also, should you be reading runtimeoptions from the project.json? I imagine that you should be doing that from the runtimeconfig.json.